xdsl/transforms/experimental/stencil_tensorize_z_dimension.py

```diff
@@ -303,20 +297,15 @@ def match_and_rewrite(self, op: dmp.SwapOp, rewriter: PatternRewriter, /):
 class StoreOpTensorize(RewritePattern):
     @op_type_rewrite_pattern
     def match_and_rewrite(self, op: StoreOp, rewriter: PatternRewriter, /):
-        if (
-            is_tensorized(op.field.type)
```

Comment:
I don't really understand why we no longer need the `is_tensorized` checks here and in `LoadOpTensorize`, is it because of the type conversion pass?

Reply:
I think this was more of a "is it actually needed and will it still run when I simply remove the checks?". Can put it back in probably, not really important.